### PR TITLE
Move support for AP_ServoRelayEvents up to GCS_MAVLINK

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by khancyr** [comment]
LGTM

File: AntennaTracker/make.inc [hunk 1]
```diff
@@ -33,3 +33,5 @@ LIBRARIES += AP_RSSI
 LIBRARIES += AP_RPM
 LIBRARIES += AP_Frsky_Telem
 LIBRARIES += AP_Beacon
+LIBRARIES += AP_Relay
+LIBRARIES += AP_ServoRelayEvents
```
Comment (OXINARF) [inline AntennaTracker/make.inc]: We should put this in the common libraries, instead of having it in all vehicles.

File: AntennaTracker/wscript [hunk 2]
```diff
@@ -9,6 +9,8 @@ def build(bld):
         ap_libraries=bld.ap_common_vehicle_libraries() + [
             'AC_PID',
             'AP_Beacon',
+            'AP_Relay',
+            'AP_ServoRelayEvents',
```
Comment (OXINARF) [inline AntennaTracker/wscript]: Same for waf.

**Comment by OXINARF** [comment]
@peterbarker Please comment on PRs, otherwise I might miss that you updated them. I've just changed last commit to have an uppercase global since I've always seen it done that way. Merged, thanks!

**Comment by peterbarker** [comment]
Ah, I thought you'd get a message for the "dismissed" review.  Learning all the time...

**Pull request closed without merging**